File: ProgramasJava/src/br/com/evertonsantos/secao06/Exercicio10.java
package br.com.evertonsantos.secao06;

import java.util.Scanner;

public class Exercicio10 {

	public static void main(String[] args) {
		int idade;
		Scanner teclado = new Scanner(System.in);
		
		System.out.println("Informe a idade do atleta: ");
		idade = teclado.nextInt();
		
		if(idade >= 5 && idade <= 7) {
			System.out.println("A categoria do atleta é Infantil A.");
		}else if(idade >= 8 && idade <= 11 ){
			System.out.println("A categoria do atleta é Infantil B.");
		}else if(idade >= 12 && idade <= 13) {
			System.out.println("A categoria de atleta é Juvenil A.");
		}else if(idade >= 14 && idade <=17) {
			System.out.println("A categoria do atleta é Juvenil B.");
		}else if(idade >= 18) {
			System.out.println("A categoria do atleta é Adulto");
		}else {
			System.out.println("O atleta não tem idade suficiente para competir!");
		}
		teclado.close();

	}

}

File: ProgramasPython/secao06/exercicio10.py
idade = int(input("Informe a idade do atleta: "))

if idade >= 5 and idade <= 7:
    print("Infantil A")
elif idade >= 8 and idade <= 11:
    print("Infantil B")
elif idade >= 12 and idade <= 13:
    print("Juvenil A")
elif idade >= 14 and idade <= 17:
    print("juvenil B")
elif idade >= 18:
    print("Adulto")
else:
    print("O atleta não tem idade suficiente para competir.")